misc/diagram-util/src/main/java/org/ballerinalang/diagramutil/DiagnosticUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.diagramutil;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import io.ballerina.tools.diagnostics.Diagnostic;
+import io.ballerina.tools.diagnostics.DiagnosticInfo;
+
+/**
+ * This is the DiagnosticUtil class for Diagnostics related Utils for the syntax tree generation.
+ */
+
+public class DiagnosticUtil {
```

Comment:
Shall we rename this to `SyntaxDiagnosticUtil` or similar since this focus on the json syntax tree generating?